open-media-match/src/OpenMediaMatch/blueprints/hashing.py

```diff
@@ -53,9 +57,21 @@ def hash_media():
     return ret
 
 
-def _parse_request_content_type() -> ContentType:
-    storage = app_resources.get_storage()
+def _parse_request_content_type(url_content_type: str) -> ContentType:
     arg = request.args.get("content_type", "")
+    if not arg:
+        if url_content_type.lower().startswith("image"):
```

Comment:
`image/` ?

Reply:
I wasn't sure if "image" by itself is a type. I didn't find any comprehensive listing.